docs/users-guide/13-embedding.md

```diff
@@ -0,0 +1,62 @@
+## Embedding Metabase in other Applications
+
+### Key Concepts
+
+#### Applications
+An important distinction to keep in mind is the difference between Metabase and the embedding application. The charts and dashboards you will be embedding live in the Metabase application, and will be embedded in your (i.e. the embedding application). 
+
+#### Parameters 
+Some dashboards, and questions have the ability to accept parameters. In dashboards, these are synnomous with dashboard filters. For example, if you have a dashboard with a filter on Publisher ID, this can be supplied as a filter when embedding, so that you could insert the dashboard filtered down to a specific Publisher ID.
+
+SQL based questions with template variables can also accept parameters for each variable. So for a query like
+```
+SELECT count(*)
+FROM orders
+WHERE product_id = {{productID}}
+```
+you could specify a specific productID when embedding the question.
+
+#### Signed parameters
+In general, when embedding a chart or dashboard, the server of your embedding application will need to sign a request for that resource. 
+
+If you choose to sign a specific parameter value, that means the user can't modify that, nor is a filter widget displayed for that parameter. For example, if the "Publisher ID" is assigned a value and the request signed, that means the frontend client that renders that dashboard on behalf of a given logged-in user can only see information for that publisher ID. 
+
+### Enabling embedding
+To enable embedding, go to the Admin Panel and under Settings, go to the "Embedding in other applications" tab. From there, click "Enable." Here you will see a secret signing key you can use later to sign requests. If you ever need to invalidate that key and generate a new one, just click on "Regenerate Key". 
+![Enabling Embedding](images/embedding/01-enabling.png)
+
+You can also see all questions and dashboards that have been marked as "Embeddable" here, as well as revoke any questions or dashboards that should no longer be embeddable in other applications. 
+
+Once you've enabled the embedding feature on your Metabase instance, you should then go to the individual questions and dashboards you wish to embed to set them up for embedding.
+
+### Embedding Charts + Dashboards
+
+To mark a given question or dashboard, click on the sharing icon 
+
+![Share icon](images/embedding/02-share-icon.png)
+
+Then select "Embed this question in an application"
+
+![Enable sharing for a question](images/embedding/03-enable-question.png)
+
+Here you will see a preview of the question or dashboard as it will appear in your application, as well as a panel that whos you the code you will need to insert in your application. 
```

Comment:
Typo: `as well as a panel that whos you the code…`
Corrected: `as well as a panel that shows you the code …`